app.js

```diff
@@ -54,10 +54,15 @@ window.addEventListener('load', () => {
             }
             await sleep(300)
         }
-        if (useJson) {
-            return fetchJson(url)
-        } else {
-            return fetchText(url)
+        try {
+            if (useJson) {
+                return fetchJson(url)
+            } else {
+                return fetchText(url)
+            }
+        } catch (err) {
+            console.log(err.message)
```

Comment:
```suggestion
            console.error(err)
```

I think that will also include the stack trace when someone looks at the message in the console. Here's an example:

![image](https://user-images.githubusercontent.com/253202/135498419-963e2472-0e4e-40ef-9547-c6657600da28.png)